go/test/query-tests/crypto/MsgNotHashedBeforeSigVerfication/MsgNotHashedBeforeSigVerfication.expected

```diff
@@ -1,6 +1,6 @@
 edges
-| MsgNotHashedBeforeSigVerfication.go:78:33:78:46 | "test message" | MsgNotHashedBeforeSigVerfication.go:78:26:78:47 | type conversion |
-| MsgNotHashedBeforeSigVerfication.go:86:31:86:44 | "test message" | MsgNotHashedBeforeSigVerfication.go:86:24:86:45 | type conversion |
+| MsgNotHashedBeforeSigVerfication.go:78:33:78:46 | "test message" | MsgNotHashedBeforeSigVerfication.go:78:26:78:47 | type conversion | provenance |  |
```

Comment:
Any idea why `provenance` was added here?

Reply:
Could be something with https://codeql.github.com/docs/codeql-overview/codeql-changelog/codeql-cli-2.10.0/#c
but I don't understand codeql's testing format for path queries